src/components/TodoAppHeader.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  allTodosCompleted: boolean,
+  disableInput: boolean,
+  titleTodo: string,
+  setTitleTodo: React.Dispatch<React.SetStateAction<string>>,
+  addHandler: () => void,
+  onToggleAll: () => void,
+};
+
+export const TodoAppHeader: React.FC<Props> = ({
+  allTodosCompleted,
+  disableInput,
+  titleTodo,
+  setTitleTodo,
+  addHandler,
+  onToggleAll,
+}) => {
+  return (
+    <header className="todoapp__header">
+      <button
+        type="button"
+        className={classNames(
+          'todoapp__toggle-all',
+          { active: allTodosCompleted },
+        )}
+        onClick={onToggleAll}
+      />
+
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
+          addHandler();
+        }}
```

Comment:
Please? check all places and create function for anonimous arrow functions in JSX